zk/conn.go

```diff
@@ -528,48 +600,58 @@ func (c *Conn) authenticate() error {
 	return nil
 }
 
-func (c *Conn) sendLoop(conn net.Conn, closeChan <-chan struct{}) error {
+func (c *Conn) sendData(req *request) error {
+	buf := make([]byte, bufferSize)
```

Comment:
Fixed in https://github.com/samuel/go-zookeeper/pull/139/commits/cc4edb777d5cf806ad064dbe2634308de162e7f6